jdk-wrapper.sh

```diff
@@ -56,7 +56,7 @@ checksum() {
     log_err "ERROR: No supported checksum command found!"
     exit 1
   fi
-  cat "${l_file}" | ${checksum_exec}
+  "${checksum_exec}" < "${l_file}"
```

Comment:
You might want `exec "${checksum_exec}"` or even simply `${checksum_exec}` (no quotes). I've noticed recently that I get this error a lot (though it doesn't seem to affect the execution of the command I'm invoking):
```
./jdk-wrapper.sh: line 59: shasum -a 256: command not found
```


Probably due to [this line](https://github.com/InscopeMetrics/kairosdb-extensions/blob/54d66d2310bda076d65bdc7ce6ced329c1da7a3b/jdk-wrapper.sh#L49).

Reply:
You are correct sir. I will make this change in jdk-wrapper.